cpu/cortexm_common/cortexm_init.c

```diff
@@ -46,7 +46,7 @@ void cortexm_init(void)
     /* set SVC interrupt to same priority as the rest */
     NVIC_SetPriority(SVCall_IRQn, CPU_DEFAULT_IRQ_PRIO);
     /* initialize all vendor specific interrupts with the same value */
-    for (int i = 0; i < CPU_IRQ_NUMOF; i++) {
+    for (int i = 0; i < (int)CPU_IRQ_NUMOF; i++) {
```

Comment:
I guess, but IRQn_Type is a signed enum below, so I wasn't sure what would be the best solution here.